### PR TITLE
Setting up support for version link issue 171

### PR DESCRIPTION
**Pull request by raffayqamar**
Contributing to issue 171:

We created a "build.properties" file and made some changes to the "build.xml" file. Within the "build.properties" file we added a site name variable, release version and the "nameofrelease". Within the "build.xml" file we added a handler for our properties file that when invoked with a keyword info, it displays the website URL, the release version, and the "name of the release."

### CODE REVIEW

**Comment by jody** [comment]
Thank you for this contribution.
Am I correct that this constitutes **part** of a resolution to User Story #171 ?
Is there an intent to continue work via this PR (keeping it open) or to close it and have future work under new PR?

**Comment by raffayqamar** [comment]
> Thank you for this contribution. Am I correct that this constitutes **part** of a resolution to User Story #171 ? Is there an intent to continue work via this PR (keeping it open) or to close it and have future work under new PR?

Issue 171 should not be closed but our work should be merged because it took care of a portion of the issue that needed to be done.

File: build.properties [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#The Site Name
```
Comment (jody) [inline build.properties:1]: Why does this one line have a comment, but no other lines?

File: build.xml [hunk 2]
```diff
@@ -6,6 +6,16 @@
   <description>
       Build file for Redistricting project investigation
   </description>
+  
+    <!--This sets up the build.properties file. Displays ant verison, site name, release verison, and name of the release-->
```
Comment (jody) [inline build.xml:10]: This comment doesn't seem appropriate. The build.properties file needs to be set up outside the automated build process. This target appears to only display the contents of an existing build.properties file.

**Review by jody — CHANGES_REQUESTED** [review]

**Comment by jody** [comment]
(No response in 17 days.)
Thank you for this contribution which prompted PR #196 that builds on this work and replaces this PR.

**Pull request closed without merging**